package.json

```diff
@@ -342,10 +342,10 @@
         "mousetrap": "^1.6.3",
         "pikaday": "^1.8.0",
         "query-string": "5.1.1",
-        "react": "^16.9.0",
+        "react": "^17.0.0",
```

Comment:
Should these also be `^17.0.1` or are we allowing the base version intentionally?

Reply:
Good catch. Fixed!